src/base/output.h

```diff
@@ -111,7 +107,23 @@ class CVC5_EXPORT Cvc5ostream
   std::ostream* getStreamPointer() const { return d_os; }
 
   template <class T>
-  Cvc5ostream& operator<<(T const& t) CVC5_EXPORT;
+  Cvc5ostream& operator<<(T const& t)
+  {
+    if (d_os != NULL)
+    {
+      if (d_firstColumn)
+      {
+        d_firstColumn = false;
+        long indent = d_os->iword(s_indentIosIndex);
+        for (long i = 0; i < indent; ++i)
+        {
+          d_os = &(*d_os << s_tab);
+        }
```

Comment:
I assume this would be more efficient (but I've not tried it...)
```suggestion
      d_os->write(s_tab.c_str(), ident);
```

Reply:
I don't think that this does what we want: `write()` writes the first `n` bytes of the first argument to the stream.